db/migrate/20190912123831_create_messages.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMessages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :messages do |t|
+      t.references  :user
```

Comment:
Same comment here, if users are not always necessary this is ok, but if not, add also `null: false`.